src/vdom/component.js

```diff
@@ -73,6 +75,11 @@ export function renderComponent(component, opts, mountAll, isChild) {
 		skip = false,
 		rendered, inst, cbase;
 
+	let getDerivedStateFromProps = component.constructor.getDerivedStateFromProps;
```

Comment:
It's usually smaller post-gzip to skip an intermediary variable like this one. Any chance that's the case here?

Reply:
Interesting, in this case it appears to not move the byte count at all. 3480 after the change (you can see it in commit b165370). So which do you prefer? Intermediary variable or not? Happy to update it to which ever.

Reply:
Spoke too soon. We can save a byte by combining this change and removing the condition in your other comment. More in a bit.